deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml

```diff
@@ -29,6 +36,33 @@ spec:
       name: clustermanagers.operator.open-cluster-management.io
       version: v1
   displayName: Cluster Manager
+  description: |
+    The Cluster Manager provides the multicluster hub, which can manage Kubernetes-based clusters across data centers, public clouds, and private clouds. This operator supports the installation and upgrade of Cluster Manager.
+
+    ## How to Install
+    Install Cluster Manager operator by following instructions in top right button `Install`. After installing the operator, create an instance of the ClusterManager resource to install the Cluster Mananger.
+
+    The pods of Cluster Manager are created in the `open-cluster-management-hub` namespace:
+
+    ```
+    $ oc get pods -n open-cluster-management-hub
+    NAME                                                       READY   STATUS    RESTARTS   AGE
+    cluster-manager-registration-controller-7c55644568-kms5c   1/1     Running   0          7h13m
+    cluster-manager-registration-controller-7c55644568-mth8f   1/1     Running   0          7h13m
+    cluster-manager-registration-controller-7c55644568-wftnj   1/1     Running   0          7h13m
+    cluster-manager-registration-webhook-77888b7dc4-bwccm      1/1     Running   0          7h13m
+    cluster-manager-registration-webhook-77888b7dc4-qk4tj      1/1     Running   0          7h13m
+    cluster-manager-registration-webhook-77888b7dc4-z9vbp      1/1     Running   0          7h13m
+    ```
+
+    There is a ManagedCluster created in the Hub cluster when the managed cluster is registried to the Hub cluster.
+
+    ```
+    $ oc get managedclusters.cluster.open-cluster-management.io
+    NAME                   HUB ACCEPTED   MANAGED CLUSTER URLS     JOINED   AVAILABLE   AGE
+    cluster-dev            true                                    True     True        7h20m
+    cluster-test           false          https://127.0.0.1:8443                        7h25m
+    ```
   icon:
   - base64data: ""
```

Comment:
@mdelder wdyt about using the open-cluster-management org icon here in the future?